src/routes/file.py

```
@@ -36,3 +38,12 @@ async def upload_image(
 @file_router.get("/file/image/download/{id}")
 async def get_image_by_id(id: str, file_service: FileServiceDep) -> FileResponse:
     return file_service.get_image_by_id(id=id)
+
+
+@file_router.get("/file/metadata")
+async def get_file_metadata(
+    file_service: FileServiceDep,
+    ids: Optional[list[str]] = Query(None, description="File IDs to fetch"),
```

Ruff (0.14.10)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)